e2e/components/UnderlineNav.test.ts

```diff
@@ -0,0 +1,159 @@
+import {test, expect} from '@playwright/test'
+import {visit} from '../test-helpers/storybook'
+import {themes} from '../test-helpers/themes'
+import {viewports} from '../test-helpers/viewports'
+
+test.describe('UnderlineNav', () => {
+  Object.keys(viewports).forEach(key => {
```

Comment:
If it helps, another option could be:

```suggestion
  Object.entries(viewports).forEach(([viewport, size])=> {
```

Reply:
Nice ✨